influxdb/tests/integration_tests.rs

```diff
@@ -140,7 +140,14 @@ async fn test_wrong_authed_write_and_read() {
             let write_result = client.query(write_query).await;
             assert_result_err(&write_result);
             match write_result {
-                Err(Error::AuthorizationError) => {}
+                Err(Error::ApiError(code)) => {
+                    if code != 403 {
+                        panic!(
+                            "Should be an ApiError(403), but code received was: {}",
+                            code
+                        );
+                    }
+                }
```

Comment:
I think this could be done more concisely, similar to the previous code:

```suggestion
                Err(Error::ApiError(code)) if code == StatusCode::UNAUTHORIZED.as_u16() => {}
```